src/@utils/order.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  approve,
+  Datatoken,
+  FreOrderParams,
+  OrderParams,
+  ProviderInstance
+} from '@oceanprotocol/lib'
+import { AssetExtended } from 'src/@types/AssetExtended'
+import Web3 from 'web3'
+import { getOceanConfig } from './ocean'
+import { TransactionReceipt } from 'web3-eth'
+import { getSiteMetadata } from './siteConfig'
+
+/**
+ * For pool you need to buy the datatoken beforehand, this always assumes you want to order the first service
+ * @param web3
+ * @param asset
+ * @param accountId
+ * @returns {TokenReceipt} receipt of the order
+ */
+export async function order(
+  web3: Web3,
+  asset: AssetExtended,
+  accountId: string
+): Promise<TransactionReceipt> {
+  const datatoken = new Datatoken(web3)
+  console.log('asset order', asset)
```

Comment:
I guess some logging would be nice but maybe we leave the logging to the component initiating the order and leave the helper method without logging? Otherwise needs to use the `LoggerInstance` here

Reply:
this is just a forgotten debug